### PR TITLE
Adventure

### PR DESCRIPTION
**Pull request by amaliahern**
DAY 5, improving fizzbuzz kata. Victor Goff thank you for your help.
Happy, happy , happy after each PR.

### CODE REVIEW

File: 2_adventures/001/amaliahern/05/fizz_buzz.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+DIVISIBLE_BY_3 = "fizz"
+DIVISIBLE_BY_5 = "buzz"
+
+def fizzbuzz(number)
+  result = number
+  if(number % 3 == 0 and number % 5 == 0)
```
Comment (kotp) [inline 2_adventures/001/amaliahern/05/fizz_buzz.rb:6]: What if we remove this line (it is checking for 3 and 5)

File: 2_adventures/001/amaliahern/05/fizz_buzz.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    result = DIVISIBLE_BY_3 + DIVISIBLE_BY_5  
+  elsif(number % 3 == 0)
```
Comment (kotp) [inline 2_adventures/001/amaliahern/05/fizz_buzz.rb:8]: And only check for 3...

File: 2_adventures/001/amaliahern/05/fizz_buzz.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    result = DIVISIBLE_BY_3 
+  elsif(number % 5 == 0)
```
Comment (kotp) [inline 2_adventures/001/amaliahern/05/fizz_buzz.rb:10]: And only check for 5...

File: 2_adventures/001/amaliahern/05/fizz_buzz.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    result = DIVISIBLE_BY_5
```
Comment (kotp) [inline 2_adventures/001/amaliahern/05/fizz_buzz.rb:11]: Because we can build that result... 

result << DIVISIBLE_BY_5

**Comment by kotp** [comment]
Be careful with the shovel though ( the shovel being the `<<`) on a string.  It changes the receiver.  This may change how you initialize your variable called `result`.

**Comment by patmaddox** [comment]
Go you!

**Pull request merged**